src/Servers/IIS/src/AspNetCoreModuleV2/CommonLib/StringHelpers.h

```diff
@@ -48,3 +51,9 @@ std::string format(const std::string& format, Args ... args)
     return result;
 }
 
+struct ignore_case_comparer
+{
+    bool operator() (const std::wstring & s1, const std::wstring & s2) const {
+        return compare_ignore_case(s1, s2) == -1;
```

Comment:
I thought compare_ignore_case would return 0 if the values were equal. Why are we comparing against -1?

Reply:
```
Binary predicate that, taking two element keys as argument, returns true if the first argument goes before the second argument in the strict weak ordering it defines, and false otherwise.
```